resources/src/main/java/org/robolectric/res/android/CppAssetManager2.java

```diff
@@ -612,7 +612,6 @@ public void ForEachPackage(PackageFunc func) {
 //                            LoadedArscEntry* out_entry, ResTable_config out_selected_config,
 //                            int* out_flags);
   private ApkAssetsCookie FindEntry(int resid, short density_override,
-      boolean stop_at_first_match,
```

Comment:
IMO, it's recommended to remove this input parameter from `FindEntry`. This method is converted from C++ asset_manager, and this input parameter will be used if we want to support `stop_at_first_match`.  Looks like caller of this method has use different value true/false for `FindEntry`.

Reply:
So, we are fine to remove it? as the caller is getting benefit from passing true/false here.

Sorry but what do you mean exactly here by
```
 Looks like caller of this method has use different value true/false for FindEntry.
 ```

Reply:
Sorry, I don't declare it correctly. IMO, I want to keep this input parameter.